### ISSUE TITLE
Spam

### ISSUE THREAD

**Issue by olivia88** [irungentoo/toxcore#1495]
Not trolling or flamebating the toxcore repo 

-- @grayhatter

**Comment by Impyy** [comment]
Not again...
This is even more inaccurate than the previous ones.

>And sorry for posting on toxcore repo

As you should be. Take this someplace else.

**Comment by optimumtact** [comment]
The neehposting is endless

**Comment by olivia88** [comment]
@Impyy 

What's inaccurate? Can you develop on your arguments?

Also, there's nowhere else to take. This is a big concern of everyone and this is the place the community has more access to.
Just look at the Tox subreddit and you'll see people demand explanations.

Don't enforce censorship by closing this! Isn't Tox about freedom of speech? Show it then. Let us talk and understand what's going on. Because as it stands right now, everybody is afraid of the situation.

**Comment by stal888** [comment]
Oh, how this reminds me of the days when I was a wee lass

**Comment by d9h02f** [comment]
https://github.com/contact?report=olivia88

**Comment by olivia88** [comment]
@d9h02f Why do that? Are you a developer? If so can you explain what's going on the channels @subliun (former Antox developer) mentioned, like #tox-secret? Because for us in the community it looks like something fishy is going on... Almost a dozen devs quit or were kicked out in the last months. How can we back up this project like this, without any explanation! Also I'm not breaking any rule so I if I stop posting here is because @irungentoo enforced censorship and blocked me.

**Comment by Zer0-One** [comment]
This poster is so disconnected from what's actually happening that I'm not sure it would even be productive to respond, except to say that you shouldn't be polluting the core repo with this garbage. Please take your hastily manufactured drama somewhere else. Closing this issue isn't censorship, it's proper maintenance.

**Comment by olivia88** [comment]
@Zer0-One What about @subliun ?
Have you read his blog post?
https://antox.me/my-time-at-tox.html
Please tell us what you have to say about that!

**Comment by LittleVulpix** [comment]
I wish people would stop posting these things on the toxcore repo. Go talk @ irc or on the dedicated neehposting github repo. Not here, this is code repo. Whether or not your concerns are valid is irrelevant, this is a repository for code. I agree with @Zer0-One - closing this is maintenance, not censorship. Go post there, not here.

**Comment by olivia88** [comment]
@LittleVulpix that repo you mentioned has 2 followers, this one has thousands. Putting this issue on a ghost repo where nobody sees it is a bad attempt of hiding it.
Please don't do this. Tox is still about freedom of speech, right? Right?

**Comment by installgen2** [comment]
![tox_jew](https://cloud.githubusercontent.com/assets/11202492/12276514/2985016e-b96e-11e5-9ea6-c18bb315d668.png)

official new tox logo

**Comment by nachfuellbar** [comment]
I'm just a user but even if this concerns should be invalid, i think this repo is the only place more people can follow this
It's sad even the second developer of antox retired from this project and nobody seem to care

**Comment by installgen2** [comment]
@nachfuellbar everyone that knew him knew he was as pictured in my previous post

**Comment by dangowrt** [comment]
@installgen2 can you elaborate more what exact properties you disliked and/or wanted to illustrate with that picture?

**Comment by installgen2** [comment]
Obsessed with centralizing Tox, would attempt to defame inner users on a daily basis, would "go on strike" whenever somebody disagreed with him, publicly sexually identified as a 12 year old, many more things which I do not have time to list now.

tl;dr
he was exactly like pic related

**Comment by olivia88** [comment]
@installgen2 our old friend and long term (2 years) Antox developer @subliun gave a lengthy, well thought and reasonable explanation about what happened. You gave us the picture of a greedy merchant? How about you address @subliun 's points and gives real answers? You're only making this worse for the foundation.

The poor guy worked for you during 2 years as the Android Tox developer and this is how talk about him? Explain yourself.

**Comment by installgen2** [comment]
@olivia88 you think jews are greedy? jesus christ, the racism here is intolerable.

**Comment by olivia88** [comment]
@installgen2 who talked about jewish people? Don't try to avoid our questions.

**Comment by installgen2** [comment]
@olivia88 can you please abide my CoC or I will forcefully remove you from my safe space, thanks

**Comment by olivia88** [comment]
Well, looks like you don't want to be taken serious.
Way to represent the Tox Foundation!

**Comment by installgen2** [comment]
when did I suggest that I represent the tox project? I stand for my CoC, and my CoC only, and you must adhere to it immediately

**Comment by nachfuellbar** [comment]
I don't think i want to follow this project anymore if this is the way to deal with concerns
Goodbye from a formerly interested user

**Comment by dangowrt** [comment]
@installgen2 you labled the images tox_jew, everyone getting email notifications has the evidence for that in their inbox. anyway, i neither care about anyones sexual preferences, nor religion, nor whatever else might annoy you about @subliun. what is obvious to me is that he and you don't seem to share a good base for communication, which is definitely not a good place to be, given you wanted to achieve something together in the past. i currently see that he is even loyal to the tox project now that he left and publicly critisized what happened -- he doesn't reveal real-identities nor unrelated personal details about anyone.

tl;dr
though power-games, accusations and all that ruin a lot, this is still a good hint that you are on the right track onto something important. if people spend that much resources to smear around, maybe time has come to first of all create a proper project consensus and resulting from that, install management structures in the way previously consented. then start re-implementing the whole thing, learning from what we already got.

**Comment by dcharles525** [comment]
This organization is cancer to the world. I guess OTR over XMPP is good enough for now.

**Comment by installgen2** [comment]
I never said that

**Comment by dangowrt** [comment]
@installgen2 you never said what? like in someone has been using my account to post stuff without me knowing about it? or are you suggesting that someone has mis-quoted you? if so, please point out and correct what was written.

**Comment by olivia88** [comment]
@dangowrt isn't that exactly what they said they would do after an ex-developer stole thousands of donated money? Looks like the situation didn't change.

@notsecure quit
@Astonex quit
@notadecent quit
@mahkoh quit
@Proplex quit
@slvr quit
@naxuroqa quit
@henotba quit
@zetok quit
@FullName quit
@urras quit
@sonOfRa quit


The list goes on...

Besides @irungentoo and a few friends he knows personally, everybody end up being betrayed by the Tox Foundation (even has money stolen or is harassed) and in the end of the day you still think nothing's wrong?

**Comment by olivia88** [comment]
How many important people have quit now? Jesus... when this many people quit don't you think it's no longer the problem of the quitters but the organization itself?

**Comment by installgen2** [comment]
I know it may seem very absurd to you, but you can see for yourself, I have not said the dreaded "w-word". I still however do not understand w*** relevance it has to this issue.

You seem to keep saying "Tox Foundation", this is already enough evidence to prove that you do not have enough relevant information to make judgments about Tox. You can read w*** happened right on the blog, it's not hard to find.

**Comment by greenbagels** [comment]
@installgen2 well if you want to play that game, you said you never said that, but by saying that, you said that.

**Comment by mahkoh** [comment]
>neehposting

lmao tbh fam

**Comment by deltabeard** [comment]
This thread is about the Tox Foundation justifying their position and actions. Please stay on topic and wait for them to answer.

This whole gossiping about who said what is not useful here. We want answers from the main developers.

> lmao tbh fam
lol get out m9

**Comment by olivia88** [comment]
@installgen2 again you didn't explain a damn thing.
I'm done playing your games. Will wait a real dev like @irungentoo come and give his explanation about what's going on.
TOX?
More like TOXic

**Comment by dangowrt** [comment]
@olivia88 I agree, the organizational problems here are obvious. imho that's no longer even worth the debate. what's at stake is the reputation of the developers involved, derived from their presumed motivation and integrity. never blame on mallice which can sufficiently be explained by stupidity (or naiivity, or average boys-club power games, or you-name-it).

**Comment by olivia88** [comment]
I fully agree with @deltabeard 

@subliun showed us his point of view = https://antox.me/my-time-at-tox.html
@Astonex showed his point of view = https://markwinter.me/my-time-tox/
@notadecent showed his point of view = https://github.com/irungentoo/toxcore/issues/1186
@urras showed his point of view through a log from irungentoo = https://archive.is/Y6LEw

Now let's see how the Tox Foundation answers.
We need an official answer, because things aren't looking good.

**Comment by installgen2** [comment]
@greenbagels it was clear that "that" was in reference to w*** as @dangowrt instantly noticed. Why don't you go back to #/g/technology you traitor of #pasta

**Comment by installgen2** [comment]
@olivia88 do you need me to spoonfeed you the link to the blog post? my fucking sides

**Comment by olivia88** [comment]
What blog post? The last one is from November about API changes.

**Comment by installgen2** [comment]
holy shit neeh it's been years and you still don't know hot to scroll down

**Comment by olivia88** [comment]
Scrolling down brings even older posts.
Are you trolling or what?
I can't believe @irungentoo let you speak in the name of his Tox Foundation.
Whatever, I'll wait for the real developers to come and discuss the issue.

**Comment by dcharles525** [comment]
so @installgen2 I am guessing this was you? https://boards.4chan.org/g/thread/52386194/the-most-successful-g-project-tox-is-being-bashed

**Comment by installgen2** [comment]
>Scrolling down brings even older posts.

jesus christ the autism
no shit it fucking does
I'm laughing so much right now

**Comment by installgen2** [comment]
@dcharles525 what would I be doing on cuckchan

**Comment by HostFat** [comment]
Is this project decentralized? Then you should **fork** it now :)

**Comment by olivia88** [comment]
@HostFat
Good question.
Is Tox really as decentralized as the developers claim?
https://github.com/irungentoo/toxcore/issues/1398

**Comment by HostFat** [comment]
@olivia88 If it isn't, is it possible to make it decentralized? What is missing? Can a fork be a good way to give it a new life?

**Comment by olivia88** [comment]
@HostFat
Fully decentralized? Possible, but comes with a price. Is it worthy it? That's up to discussion.

What is missing? Organization. The Tox Foundation spies on its old users (like Proplex, see https://archive.is/Y6LEw for details), harass others (see @subliun @astonex and @notadecent), steals donation money (see what @stqism) and all that.

If a fork can give it new life? I don't know. Most security experts I talked to think it's fundamentally messy and hard to deal with, so they aren't interested. But if a professional fork appeared, maybe it could give new life. I'd be eager to make a Client for it myself, depending on the new organization (and if it was 100% new). But as it is, the situation looks **bad**. The Foundation is hugely unethical in the eyes of the community and they don't do anything to explain themselves.

**Comment by JFreegman** [comment]
Please don't tag me in your troll threads neeh.

**Comment by dangowrt** [comment]
@installgen2 you didn't explain anything, there was nothing I could have noticed others couldn't.
@HostFat @olivia88 regarding *technical* decentralization: the thing to review here is the election and choice supernodes. some sort of NAT traversal using TCP and actually globally IPv4-reachable nodes is unavoidable when wanting to offer a service on todays NAT'ed IPv4-only mobile internet access services. (remember? you always had to "open up those ports" to get P2P stuff working properly. On a mobile connection, there is just no way to do that.) There is no other way to get end-to-end communication, and this issue has been discussed as part of most other (mobile) P2P development efforts.

I highly recommend everyone interested in an honest debate to subscribe to receive email notifications, especially for heated debates like this. I say this because posts on github can be edited by the people who posted them without leaving any trace of the changes (very unlike the rest of github) visible on the web-ui. When subscribing to email notifications, you at least get to see what was originally said (and replied to) and don't end-up with gibberish...

@olivia88 The only interesting thing about Tox is portability and popularity. Both are not amazing, but notable. And well, some of the code is useful, like UI and multimedia stuff. If you want to start a new (mobile) secure p2p communication client, please base it on something more decent, so your efforts won't be a waste of time. Tox was nice draft and I reckon we all have learned a lot from it.

**Comment by mahkoh** [comment]
>https://antox.me/my-time-at-tox.html



>stal and zero-one voted against me being on the committee simply because they did not like me personally, Jfreegman voted against me because my ideas were different from his (he told me this himself when I asked about it)



>Every major client developer who requested an email was given a @tox.chat email address, bar me.

neehposting aside, why do you have to bully people so hard that they start to blog about it? This is really mean if it's true.

**Comment by JFreegman** [comment]
@mahkoh It's not true, he seems to have a victim complex. He was voted against because of his immature behaviour and the fact that he wants to centralize Tox. And we don't give tox.chat emails out freely to anyone who asks. Tux3 doesn't even have one.

**Comment by olivia88** [comment]
@JFreegman but did @tux3 ask for one?
Was any developer with more than 1 year working on a client denied an email?

**Comment by JFreegman** [comment]
We actually gave one to IRIXuser then had to take it away from him when we realized he could read organization emails in our catchall.

**Comment by optimumtact** [comment]
I don't think those emails are even for anyone who asks, they're for voting group/infrastructure users

**Comment by olivia88** [comment]
To be honest why did you want an email @subliun ?
It's a known fact that irungentoo and his friends from #tox-secret spies on other devs emails.
@Proplex was a victim in the past. https://archive.is/Y6LEw
Props to @urras for revealing this unethical behavior of the Tox Foundation.

Anyway, this is all relevant and all, but it looks like we aren't getting any answers here...
Will the Tox Foundation answer to @subliun and @Astonex complains?

**Comment by stqism** [comment]
@olivia88 Still at it neeh? Jesus, you pick a new name every bit of drama to make these shitty posts as a new person. 

Don't you have something better to do than shitpost on the internet your entire life? You know, work or school? No? of course not.

Could you please crawl out of your fucking hole in Brazil and do something with your life that doesn't involve mentioning me? I get a new email every damn time.

**Comment by olivia88** [comment]
@stqism You got the wrong person, but since you are here, what's your opinion on the Tox Foundation?

**Comment by installgen2** [comment]
@olivia88 You still seem to be misunderstanding, he IS the Tox foundation, and aside from @stqism, IT IS NOTHING ELSE.

**Comment by suhr** [comment]
> @subliun and @zetok kick

Oh hell. What have I missed so far?

**Comment by olivia88** [comment]
Tox Foundation, Tox Organization, whatever you call it.

The ones that make the final decision and have their secret channels to harass others/steal money/etc. As we saw in the past. https://archive.is/Y6LEw

I know @irungentoo is part of it together with @Zer0-One so an explanation from them would be good about this whole @subliun situation.

**Comment by stqism** [comment]
@olivia88 Neeh, it's a corporate shell based out of Los Angeles, CA that served its purpose and then fell in to obscurity.

#tox-sercret isn't the Tox Foundation you half-wit.

**Comment by olivia88** [comment]
@stqism How do we call the Tox leaders? @irungentoo @Zer0-One etc ?

**Comment by installgen2** [comment]
call them "shitters"

**Comment by stqism** [comment]
I'd call them @irungentoo and @Zer0-One but apparently that's too difficult for you.

**Comment by optimumtact** [comment]
I don't see why making a decision to not focus on setting up an fdroid repo or to run a name look up service is harassing others.

We had other more important infrastructure to focus on and limited time and money to achieve results

**Comment by CRImier** [comment]
Oh wow, and I was developing a client. Better wait until everything is clear between you all guys.

**Comment by LuccoJ** [comment]
@JFreegman Regardless of all the other issues, the maintainer of the one "official" client for one of the most important architectures used by people is hardly "just anyone who asks".
I think giving @subliun this sort of appellation in public kind of confirms that he was being treated as inferior to other client developers.

**Comment by LuccoJ** [comment]
@optimumtact It's easy to have limited money when the money you are supposed to have was stolen by an ex-member, isn't it? ;-)

**Comment by optimumtact** [comment]
Yes well there's nothing I can do about that now is there, and it's rude of you to insinuate that it's somehow my fault when I've busted my ass off to get infrastructure back together out of pocket.

**Comment by JX7P** [comment]
@olivia88, yours is a fantastic concoction of half-truths, fabrications, and insults. It is a wonder I am taking the time to reply to someone who has made clear their determination to stampede over anything Tox ever does. It is a testament to the patience of Tox developers and maintainers that anyone is responding to this abhorrent provocation.

>The Tox developer @subliun stopped working for Tox after he was disrespected.

A lot of people claim to have been disrespected. @subliun went so far as to disrespect me in their blogpost, belittling my extensive efforts on XwinTox. I won't pretend that everyone gets along fine, but generally things go well unless people decide that they must be constantly complimented. Then there are problems.

>@Astonex was another developer that quit a long time ago too.

He is also well-known for his antisemitic and racist comments.

>The Tox leader @irungentoo has total control over the organization according to the law (tox is registered on the government). He and his personal friends put on charge, like @Zer0-One and @JFreegman are very toxic for the open-source movement and are trying to (at least that's how I see) profit from Tox for themelves. Over and over they've been warned about @stqism and others

This is a fantasy. No one 'warned' anyone about @stqism or, for that matter, about anyone else. And if you think anyone imagines profit in Tox, I suggest you try listening to the people who are doing their best to serve the community. People like those you listed. As to @Zer0-One and @JFreegman, zero-one has a proven track record in community leadership, and JFreegman is the major force behind one of the first and most popular Tox clients.

>Besides, the security concerns were IGNORED for a long time. They continue to advertise Tox as secure even though it's still alpha. This is UNETHICAL and people are getting hurt. Many developers saw this and quit, others were kicked from the project. But in the end someone has to stand out and reveal how toxic this has become.

Tox is a Secure, Free, Fun Messenger. It is a scientific fact that Tox is secure. No one is getting 'hurt' at all. This is fantasy.

>What the community needs now is an explanation from the developers. What's the deal? What's going on? Why kick @subliun and @zetok like that? Are everyone in the Tox Foundation fine with that? Is @tux3 aware of what's going on? The community is the one to suffer.

subliun was not kicked out. They left voluntarily.

>We are losing faith in Tox! How can we trust all this? [...] personal information from users leaked

No such incident has ever occured.

>And sorry for posting on toxcore repo, but this is obviously the only place the community follows and this is a HUGE concern of everybody, so let's discuss the matter like adults and not enforce censorship by closing the issue, ok

My personal preference is for the deletion outright of this nefarious provocation.

**Comment by deltabeard** [comment]
>No such incident has ever occured.

I guess that settles it. This (and some other) misinformation was what had me concerned. As far as I'm concerned, this conversation is over.
Maybe just lock this thread in case someone asks the same thing in the future?

Also @stqism, you're comments here aren't too friendly, yet wasn't it yourself that caused most of this mess?

**Comment by nurupo** [comment]
Are you the same person who created similar issues last year? Because you write almost the exact things, almost repeating them line to line, as inaccurate as that person did.

Just take that "Tox Foundation" nonsense you are talking about. There is no "Tox Foundation" anymore, and as far as I know, stqism was the sole member of "Tox Foundation" (though there was no proof) and he is not working on anything Tox anymore.

Also that list of quit developers is artificially inflated, as some of them never actually "joined" in the first place in order to quit (passer by contributors, GSoC students, etc.), while other from the list are not contributing simply because they are busy with their daily life, like @sonOfRa (he is even a channel operator in #tox-dev, why do you think he would be an op if he has quit?), whom we expect to get back to us. A part of the list is accurate though, some of the people you listed did quite because they didn't like the community and how things were done, but that's the point -- you combine a few accurate facts and add a lot of misinformation on top of it to support your claims and to make it look like it's a big mess.

I'm not going to pointing out every single inaccuracy you have made, as I'm quite sure you did it on purpose to provoke developers, so I would just waste time doing that. Please take your nonsens somewhere else, stop harassing the community.

**Comment by JFreegman** [comment]
@LuccoJ Being a client developer doesn't entitle you access to organizational infrastructure. Most client developers do not have @tox.chat emails and his claim that he was the only one who was denied an email is patently false.

**Comment by olivia88** [comment]
@JX7P

> A lot of people claim to have been disrespected

This just shows how toxic the tox developers are.

> @subliun went so far as to disrespect me in their blogpost, belittling my extensive efforts on XwinTox

Well, I also thought XwinTox was a joke. I'm sorry if it isn't, but the way you talk and act implied it in the past. I suggest you try acting more professionally. Does anyone even use XwinTox? Antox, on the other hand, was used by thousands of people.

> zero-one has a proven track record in community leadership

Now you are just trolling. Did you read subliun's blog post? here's the link for you - https://antox.me/my-time-at-tox.html
zero-one kicked zetok for no reason and destroyed his STS voting system where all client developers decided on the standard. Instead, he and JFreegman made all decisions themselves. Nice leadership!

> Tox is a Secure
Prove it. You keep advertising alpha software with no audit as secure. This is unethical and people are being deceived. Say it's "probably" secure or something, but don't make these bold claims without certainty. 

> It is a scientific fact that Tox is secure. No one is getting 'hurt' at all. This is fantasy.

Oh wow. You're either deluded or trolling. Show me the "scientific" fact that proves Tox is secure.

> subliun was not kicked out. They left voluntarily.

Being kicked from the channel, denied the infrastructure you need and harassed a "voluntary quit"?

> No such incident has ever occured.

Personal information **did** leak.
Specifically an old contributor's resume, IRC logs showing notsecure was irungentoo's brother (you leaked this one, actually), Proplex user-agent ( https://archive.is/KkSWp ) etc.

**Comment by mahkoh** [comment]
@JX7P

>He is also well-known for his antisemitic and racist comments.

Please don't bully Astonex. It's much better to be nice to each other.

![image](https://cloud.githubusercontent.com/assets/1882250/12280175/40ec5776-b98c-11e5-8edb-21fcab04cbf6.png)

**Comment by suhr** [comment]
@olivia88 I know you're yet another dumb troll (probably the same who creates issues before), but thank you for posting https://antox.me/my-time-at-tox.html. It was the only good thing you ever did.

Well, tox folk, you've finely deserved my distrust.

**Comment by LuccoJ** [comment]
@JX7P Security is a journey, not a destination. Anyone claiming that a piece of software is "scientifically secure" would be automatically discredited by the entire security community.

**Comment by stqism** [comment]
@deltabeard 
>Also @stqism, you're comments here aren't too friendly, yet wasn't it yourself that caused most of this mess?

I dropped my nice and friendly PR filter when I left the project, but the fact of the matter is that @olivia88 is some moron who's been trolling the project for a solid year now.

**Comment by olivia88** [comment]
You're making a lot of claims about @subliun blog post being a lie, @JFreegman 
I say we wait for his response. He should be able to defend himself.

**Comment by nikitasius** [comment]
have a popcorn, read all tomorrow!

**Comment by LuccoJ** [comment]
@optimumtact I have't insinuated a thing, but I'm pretty sure you insinuating and/or believing that I have and calling it rude fits very well in the tone of this discussion and Tox atmosphere in general at this point.

**Comment by optimumtact** [comment]
well perhaps I misread the tone of your comment, but it looked condescending to me

**Comment by deltabeard** [comment]
@stqism 
@olivia88 may be some moron, but did you not take money from donations made towards the Tox project?
If so, why are you here talking about someone else's poor behaviour, when you yourself should be hanging your head in shame?

**Comment by dangowrt** [comment]
@JX7P claiming 'security' will lead people to believe they should use Tox to share info online they would otherwise keep offline or not communicate at all. Not having any warning signs on a codebase as young and exotic as this is irresponsible imho. Saying "this is beta-quality software. don't trust it!" would help the reputation of this project in the FOSS community and make it a good example on how (despite the "no warranty" term in the license) to go about social responsibility of crypto-stuff.

@nikitasius make sure to get email-notifications on posts (because that way people can't revoke or change them) and lots of popcorn for breakfast then.

And wow, you all actually got enemies and communication hickups to deal with... Entertaining, but certainly not productive or helpful.

**Comment by wiiaam** [comment]
@dangowrt your profile picture seems to be missing friend

**Comment by LuccoJ** [comment]
@optimumtact It was more like, "look what a whole mess this is". Maybe it's condescending, but not to you personally, I don't even known you.
For sure, it would be best for people (still) actively involved with this project to recognize the mess is not reflecting well on the project. Sometimes one thinks the world is just full of trolls and it all just reflects badly on the trolls, but well, no, that's not what happens. "Little mistakes" like the money loss, with people calling out the signs being ignored and so on... didn't help, and may partake in causing a burst like we are witnessing.

**Comment by stqism** [comment]
@deltabeard Because I accepted what happened and put it aside to do my own thing, leaving the project to be except when mentioned.

@olivia88 is a troll who waits for drama to make these posts 24/7.

Also, 
> did you not take money from donations made towards the Tox project? 
is FUD.

Donations went to overpriced servers, stickers, etc. A lot of the time every month we didn't actually break even paying infrastructure costs because I was an idiot. What the whole drama was about was because when it came down to it we had almost nothing left from our constantly draining budget. The majority of the money came from GSoC addressed to the Tox Foundation and not Tox project donations regardless.

**Comment by JX7P** [comment]
>This just shows how toxic the tox developers are.

It shows how a community as large as that of Tox does not always run smoothly.

>Well, I also thought XwinTox was a joke. I'm sorry if it isn't, but the way you talk and act implied it in the past. I suggest you try acting more professionally. Does anyone even use XwinTox? Antox, on the other hand, was used by thousands of people.

This says all I need to know. XwinTox was formed for several purposes, but joking was not one of them. XwinTox enjoys a user community that may not equal that of Antox in size, but which remains nonetheless significant.

>Now you are just trolling. Did you read subliun's blog post? here's the link for you - https://antox.me/my-time-at-tox.html
>zero-one kicked zetok for no reason and destroyed his STS voting system where all client developers decided on the standard. Instead, he and JFreegman made all decisions themselves. Nice leadership!

Subliun is already known to have personal issues with zero-one.
The STS fell into lack of maintainership after the departure of @stqism. Plans were being made for months on a successor. The decisions made by the TCS committee are not taken by JFreegman and zero-one, but by the committee.


>Prove it. You keep advertising alpha software with no audit as secure. This is unethical and people are being deceived. Say it's "probably" secure or something, but don't make these bold claims without certainty.

Audits are not the way free software is proven secure. In strict terms, actually, it is difficult to prove anything secure. Audits are very expensive, do you know why? It is because they are used for proprietary software, and come with a powerful guarantee against the release of source code to third parties. I favour further safeguards on Tox development to ensure further that security is kept ever-in-mind, but that's tangential. In the meantime I will present a project which takes as its utmost priority security as being a secure project.

>Oh wow. You're either deluded or trolling. Show me the "scientific" fact that proves Tox is secure.

It's polemic in response to yours.

>Being kicked from the channel, denied the infrastructure you need and harassed a "voluntary quit"?

Kicked from what's often called the 'meme channel', because it's where a certain community of people who know each other well stay, not where a secret cabal lays its devilish plans for world domination.

>Personal information did leak.
>Specifically an old contributor's resume, IRC logs showing notsecure was irungentoo's brother (you leaked this one, actually), Proplex user-agent ( https://archive.is/KkSWp ) etc.

That would be yours. It wasn't leaked, either - it was provided to relevant persons after you began your anti-Tox vendetta. It was important that you were dealt with, and you were dealt with in perhaps the mildest manner I can imagine. A user-agent does not really constitute personal information, and the IRC logs I provided were released during my own efforts to promote transparency in Tox development. I succeeded.

>@JX7P Security is a journey, not a destination. Anyone claiming that a piece of software is "scientifically secure" would be automatically discredited by the entire security community.

I'm much more of a polemicist than someone who takes as their guiding principle the exact correctness of terminology. Tox is on the secure journey and always has been. 'Scientifically secure' is a fantastically ludicrous statement and could only be interpreted as counterinvective against provocation.

>@JX7P claiming 'security' will lead people to believe they should use Tox to share info online they would otherwise keep offline or not communicate at all.

Unfortunately it really isn't that simple. That claim is provided without evidence. Personally I favour the view that people will use Tox instead of Skype or some similar proprietary solution. In which case I sincerely hope that they do use Tox to share that information.

>Not having any warning signs on a codebase as young and exotic as this is irresponsible imho. Saying "this is beta-quality software. don't trust it!" would help the reputation of this project in the FOSS community and make it a good example on how (despite the "no warranty" term in the license) to go about social responsibility of crypto-stuff.

I advocate Tox's use for secure communication on the grounds that it is certainly more secure than Skype, though with the proviso that PGP or other longer-tested techniques may likely be superior to Tox. It'd be madness to work on a secure messenger and use fear-inducing phrases like "do not trust this" which may well motivate the user to simply use Skype, Facebook Messenger, or whatever other certainly-insecure messenger they may use if they do not see something likely more secure.

**Comment by GrayHatter** [comment]
So, there's no trolling this Repo's issues.

I've deleted a lot of posts that are trying to link mine, or are dox level links that violate GH tos. And a few that are mostly half truths. 

If you think this is a mistake. Pm me on IRC and convince me otherwise. But toxcore isn't the place for this kinda trolling, and or FUD.

Also, if your GH account is less than a day old, you can go f*** yourself. I'll ignore you for the troll you are @olivia88 people might like you more if you wrote code instead of FUD. (Or at least listen.)  But even that won't make daddy love you.

**Comment by GrayHatter** [comment]
So, there's no trolling this Repo's issues.

I've deleted a lot of posts that are trying to link mine, or are dox level links that violate GH tos. And a few that are mostly half truths. 

If you think this is a mistake. Pm me on IRC and convince me otherwise. But toxcore isn't the place for this kinda trolling, and or FUD.

Also, if your GH account is less than a day old, you can go f*** yourself. I'll ignore you for the troll you are @olivia88 people might like you more if you wrote code instead of FUD. (Or at least listen.)  But even that won't make daddy love you.

**Comment by GrayHatter** [comment]
So, there's no trolling this Repo's issues.

I've deleted a lot of posts that are trying to link mine, or are dox level links that violate GH tos. And a few that are mostly half truths. 

If you think this is a mistake. Pm me on IRC and convince me otherwise. But toxcore isn't the place for this kinda trolling, and or FUD.

Also, if your GH account is less than a day old, you can go f*** yourself. I'll ignore you for the troll you are @olivia88 people might like you more if you wrote code instead of FUD. (Or at least listen.)  But even that won't make daddy love you.

**Comment by GrayHatter** [comment]
So, there's no trolling this Repo's issues.

I've deleted a lot of posts that are trying to link mine, or are dox level links that violate GH tos. And a few that are mostly half truths. 

If you think this is a mistake. Pm me on IRC and convince me otherwise. But toxcore isn't the place for this kinda trolling, and or FUD.
